### PR TITLE
Add default CircleCi configuration

### PR DESCRIPTION
**Pull request by johnpaulashenfelter**
## Description of change

Evaluating CircleCI build process for vets-api.

#### Unique to this PR

- [ ] _Replace this line with individual tasks unique to your PR_

#### Applies to all PRs

- [ ] Appropriate logging
- [ ] Swagger docs have been updated, if applicable
- [ ] Provide link to originating GitHub issue, or connected to it via ZenHub
- [ ] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [ ] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2 
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.3-stretch-node
```
Comment (johnpaulashenfelter) [inline .circleci/config.yml]: I know this is slightly different than our current one. Digging into the docs to see what our options are.

**Comment by annaswims** [comment]
all PR's now show " Your tests failed on CircleCI" should we unto the status check until this is merged?

**Comment by omgitsbillryan** [comment]
Is there a ticket this is connected to? Also, any additional context you could provide as to why we're doing this would help greatly.

**Comment by johnpaulashenfelter** [comment]
@omgitsbillryan issue is upcoming. readout from devops was heading this way, frontend is trying as well. seems like a small lift but this is acting weird.

**Comment by johnpaulashenfelter** [comment]
Closing because there's no easy way to do in *this* repository without breaking deploys (review instance deploys at least). Jenkins expects *all* checks to be passing and since this CircleCi check will be broken until it works, AND since every branch gets the same set of checks including CircleCI we end up in a broken situation.

The right way to do this is figure out the issues on a fork

**Pull request closed without merging**